--- ima/src/classroom_manager.cpp.str	2010-12-05 12:12:00.000000000 +0000
+++ ima/src/classroom_manager.cpp	2010-12-05 12:12:25.000000000 +0000
@@ -428,13 +428,13 @@
 	QString xml = "<?xml version=\"1.0\"?>\n" + doc.toString( 2 );
 	if( MainWindow::ensureConfigPathExists() == FALSE )
 	{
-		qWarning( QString( "Could not read/write or create directory "
+		qWarning() << QString( "Could not read/write or create directory "
 					"%1! For running iTALC, make sure you "
 					"have write-access to your home-"
 					"directory and to %1 (if already "
 					"existing)."
 				).arg( localSystem::personalConfigDir()
-						).toUtf8().constData() );
+						).toUtf8().constData();
 	}
 
 	QFile( m_personalConfiguration + ".bak" ).remove();
