 lib/src/lock_widget.cpp |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/lib/src/lock_widget.cpp b/lib/src/lock_widget.cpp
index 7989ed8..99e2d2b 100644
--- a/lib/src/lock_widget.cpp
+++ b/lib/src/lock_widget.cpp
@@ -68,6 +68,7 @@ lockWidget::lockWidget( types _type ) :
 	m_sysKeyTrapper.disableAllKeys( TRUE );
 	setWindowTitle( tr( "screen lock" ) );
 	setWindowIcon( QIcon( ":/resources/icon32.png" ) );
+	setWindowFlags( Qt::Popup );
 	setCursor( Qt::BlankCursor );
 	showFullScreen();
 	move( 0, 0 );
