--- acinclude.m4.orig	2010-12-05 12:01:15.000000000 +0000
+++ acinclude.m4	2010-12-05 12:00:47.000000000 +0000
@@ -127,15 +127,16 @@
 	fi
         ;;
     *)
-       	QT_LIBS=`ls $QTDIR/lib64/libQt*.so 2> /dev/null` 
+	QT_LIBDIR=`pkg-config QtCore --variable=libdir`
+       	QT_LIBS=`ls $QT_LIBDIR/libQt*.so 2> /dev/null` 
         if test "x$QT_LIBS" = x;  then
-        	QT_LIBS=`ls $QTDIR/lib/libQt*.so 2> /dev/null` 
+        	QT_LIBS=`ls $QT_LIBDIR/lib/libQt*.so 2> /dev/null` 
             	if test "x$QT_LIBS" = x;  then
 			AC_MSG_ERROR([*** Couldn't find any Qt4 libraries])
             	fi
-		QT_LIB="-L$QTDIR/lib -L$QTDIR/lib/qt4"
+		QT_LIB="-L$QT_LIBDIR"
 	else
-		QT_LIB="-L$QTDIR/lib64 -L$QTDIR/lib64/qt4"
+		QT_LIB="-L$QT_LIBDIR"
         fi
  	QT_LIB="$QT_LIB -lQtCore -lQtXml -lQtNetwork"
 	QT_LIB_GUI="-lQtGui"
