--- ima/data/italc.desktop.in	2007-06-09 03:26:10.000000000 +0530
+++ ima/data/italc.desktop.in.new	2008-08-11 19:27:54.000000000 +0530
@@ -5,7 +5,7 @@
 GenericName[de]=Clients überwachen und fernsteuern
 Comment=Intelligent Teaching And Learning with Computers
 Icon=@prefix@/share/pixmaps/italc.png
-Exec=@prefix@/bin/italc
+Exec=@prefix@/bin/italc-launcher
 Type=Application
 Terminal=false
 Categories=Qt;KDE;Education;X-KDE-Edu-Teaching;
